internal/commands/manifest_rm.go

```diff
@@ -1,44 +1,38 @@
 package commands
 
 import (
-	"path/filepath"
-
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 
-	"github.com/buildpacks/pack/internal/config"
-	"github.com/buildpacks/pack/internal/style"
-	"github.com/buildpacks/pack/pkg/client"
 	"github.com/buildpacks/pack/pkg/logging"
 )
 
+// ManifestRemove will remove the specified image manifest if it is already referenced in the index
 func ManifestRemove(logger logging.Logger, pack PackClient) *cobra.Command {
+	// var flags ManifestRemoveFlags
+
 	cmd := &cobra.Command{
-		Use:   "rm [manifest-list] [manifest]",
-		Short: "Remove an image manifest from index",
-		Args:  cobra.MatchAll(cobra.ExactArgs(2)),
+		Use:   "rm [manifest-list] [manifest] [manifest...] [flags]",
+		Args:  cobra.MatchAll(cobra.MinimumNArgs(2), cobra.OnlyValidArgs),
+		Short: "manifest rm will remove the specified image manifest if it is already referenced in the index",
 		Example: `pack manifest rm cnbs/sample-package:hello-multiarch-universe \
-					cnbs/sample-package:hello-universe-windows`,
-		Long: "manifest remove will remove the specified image manifest if it is already referenced in the index",
+		cnbs/sample-package:hello-universe-windows`,
```

Comment:
I think we should update the example and the documentation to include that the *digest* is required. 

I got errors like 

```bash
a digest must contain exactly one '@' separator (e.g. registry/repository@digest) saw: cnbs/sample-package:hello-universe-windows
```
And need to look for it

Reply:
now pack cli supports removing images by tags along with digests